src/ci/docker/x86_64-gnu-llvm-7/Dockerfile

```diff
@@ -26,7 +26,7 @@ ENV RUST_CONFIGURE_ARGS \
       --build=x86_64-unknown-linux-gnu \
       --llvm-root=/usr/lib/llvm-7 \
       --enable-llvm-link-shared
-ENV SCRIPT python2.7 ../x.py test src/tools/tidy && python2.7 ../x.py test
+ENV SCRIPT python3 ../x.py test src/tools/tidy && python3 ../x.py test
```

Comment:
Can we keep this one on python2.7, to make sure that continues to work?

Reply:
I agree this would be good.